build.sbt

```diff
@@ -57,13 +58,13 @@ daemonGroup in Docker := dockerGroup
 dockerBaseImage := "develar/java"
 
 dockerCommands := dockerCommands.value.flatMap {
-  case cmd@Cmd("WORKDIR", _) => List(cmd,
-    Cmd("RUN", installAll)
-  )
-  case cmd@(Cmd("ADD", "opt /opt")) => List(cmd,
-    Cmd("RUN", "mv /opt/docker/docs /docs"),
-    Cmd("RUN", s"adduser -u 2004 -D $dockerUser"),
-    ExecCmd("RUN", Seq("chown", "-R", s"$dockerUser:$dockerGroup", "/docs"): _*)
-  )
+  case cmd @ Cmd("ADD", _) =>
+    List(
+      Cmd("RUN", s"adduser -u 2004 -D $dockerUser"),
+      cmd,
+      Cmd("RUN", installAll),
+      Cmd("RUN", "mv /opt/docker/docs /docs"),
+      ExecCmd("RUN", "chown", "-R", s"$dockerUser:$dockerUser", "/docs")
```

Comment:
Here you can use Cmd and a single string as in the command above

Reply:
Done ✅